src/MLJGLMInterface.jl

```diff
@@ -85,6 +85,9 @@ function augment_X(X::Matrix, b::Bool)::Matrix
     return X
 end
 
+_to_vector(v::Vector) = v
+_to_vector(v) = collect(v)
+
```

Comment:
```suggestion
```
Calls to `_to_vector` can be replaced by `collect`. 
```julia
julia> [1, 2] isa Vector
true

julia> collect([1, 2])
2-element Vector{Int64}:
 1
 2
```

Or is there are reason why this should be a separate function?

Reply:
Yeah, allocation